rpcs3/Crypto/unself.cpp

```diff
@@ -1316,7 +1326,7 @@ bool CheckDebugSelf(const std::string& self, const std::string& elf)
 
 		// Copy the data.
 		char buf[2048];
-		while (ssize_t size = s.read(buf, 2048))
+		while (size_t size = s.read(buf, 2048)) // Did size need to be of type ssize_t?
```

Comment:
I'm rewriting it so it will not return -1 anymore, so just forget about this detail. If the u64 is returned, then it must be neither ssize_t nor size_t, but u64.